tests/Feature/CelebrityTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Models\User;
+
+class CelebrityTest extends TestCase
+{
+    /** @test  */
+    public function test_users_can_browse_celebrity()
+    {
+        $user = User::factory()->create();
+
+        $this->actingAs($user)->get('/celebrities')
+                              ->assertOk();
```

Comment:
Usually when we have a sequence that has `::` or `->`. We format it in a way where each statement after the first one stays in its own line. 
```
$this->actingAs.... 
       ->get()
       ->assert 
```
Something like this.